java/shims/pom.xml

```diff
@@ -42,6 +42,10 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.github.rdblue</groupId>
+      <artifactId>brotli-codec</artifactId>
+    </dependency>
```

Comment:
Sorry, but this looks wrong to me. Please do not add the compression dependency to `shims` module.

Reply:
Got it, let me try to re-implement this part.